public/app/features/plugins/admin/types.ts

```diff
@@ -322,3 +322,7 @@ export type InstancePlugin = {
   pluginSlug: string;
   version: string;
 };
+
+export type ProvisionedPlugin = {
+  type: string; // type contains the plugin slug
```

Comment:
nit: it might be worth to to rename type to slug so it is more explicit in the frontend code. Then use a inline type when fetching the data.